README.md

```diff
@@ -934,7 +934,18 @@ Now `field` is available as a local property of the template and can be invoked
 {{ field('pass', type='password') }}
 ```
 
-> Macros need to be defined before they are used as they are discovered as the template is executed. They can also be defined in external templates and imported using the `{% include %}` tag.
```

Comment:
The PR is not changing the behavior wrt includes, so this will still work, right? Not saying this should be removed. I also remember that `include` is not recommended in Liquid (because like in this case it leaks to the caller).

Reply:
>The PR is not changing the behavior wrt includes, so this will still work, right?
Yes behavior of includes isn't changed by this PR. But I would still say that include can't be used to add external macros to the local scope. The following example won't work:

functions.liquid
```
{%- macro hello_world() -%}
Hello world!
[%- endmacro -%}
```

template.liquid
```
{%- include 'functions' %}
{{ hello_world() }}
```

This will **not** render `Hello world!`

Actually the opposite will work.
You can access a marco declared in a template from the included file.

Hope this answers your question.